src/debugging/coreclr/CommandLineDotNetClient.ts

```diff
@@ -48,6 +60,75 @@ export class CommandLineDotNetClient implements DotNetClient {
             return undefined;
         }
     }
+
+    public async trustAndExportSslCertificate(projectFile: string, hostExportPath: string, containerExportPath: string): Promise<void> {
+        if (CommandLineDotNetClient.KnownConfiguredProjects.has(projectFile)) {
+            return;
+        }
+
+        await this.addUserSecretsIfNecessary(projectFile);
+        await this.promptAndTrustCertificateIfNecessary();
+
+        const password = uuidv4();
+
+        // Export the certificate
+        const exportCommand = `dotnet dev-certs https -ep "${hostExportPath}" -p "${password}"`;
```

Comment:
FWIW, we do this in VS. Generally probably not necessary--but, we can't verify if the password has been set (without another commandline), nor can we verify that the cert matches the password.

Reply:
But why would the password or cert be incorrect after the initial time we set it? If it's changed - doesn't that imply the user changed it? That sounds like the intentional action of an advanced user (aka hard to do "accidentally") and I would prefer to defer to them rather than assume we know better.

Even though you say it doesn't cost much (and I believe you) - I'm also just very hesitant to run things every F5. It's a high occurrence scenario that you want to be as fast and reliable as possible.

Reply:
Yes, monkeying with it would mess things up. Also, it's not every F5, it's first F5 of the session.

Reply:
> Yes, monkeying with it would mess things up.

Right I'm saying if the user messes things up, I'm fine with that. I generally don't trust us to get it right for every single combination of OS, .NET, etc. In other words, I'm saying some users might have successfully configured SSL for ASP.NET and we actually end up breaking them by running this every session. If we decide "users can turn it off in their launch config" is the answer, I guess that's fine. It just feels unnecessary to run every session

> Also, it's not every F5, it's first F5 of the session.

True. Although if you're code isn't perfect (I know - just a hypothetical) and throws an error before `CommandLineDotNetClient._KnownConfiguredProjects.add(projectFile)` it will end up being "every F5"